akvo/templates/organisation_main.html

```diff
@@ -123,47 +123,67 @@ <h4 class="topMargin">{% trans 'IATI file' %}</h4>
             </a>
           </h2>
           <dl class="dl-horizontal">
-            <dt>{% trans 'Active' %}</dt>
+            <dt class="iatiStatuses">{% trans 'Implementation' %}</dt>
             <dd>
               {% if organisation.published_projects.status_active.count %}
-                <a href="{% url 'project-directory' %}?organisation={{ organisation.pk }}&amp;status=A">
+                <a href="{% url 'project-directory' %}?organisation={{ organisation.pk }}&amp;iati_status=2">
```

Comment:
In this list I would put status 1 (Pipeline/identification) first, then status 2 (Implementation).